apps/api/src/workspace/services/workspace.service.ts

```diff
@@ -418,7 +418,11 @@ export class WorkspaceService {
       throw new NotFoundException(`Workspace with ID ${workspaceId} not found`)
     }
 
-    if ([WorkspaceState.DESTROYED, WorkspaceState.UNKNOWN, WorkspaceState.CREATING].includes(workspace.state)) {
+    if (
```

Comment:
This can be removed completely since we have `validateDesiredState` in the workspace entity model.